lib/presentation/extensions/send_file_extension.dart

```diff
@@ -308,11 +308,21 @@ extension SendFileExtension on Room {
     required String fileName,
   }) async {
     try {
-      final filePathInAppDownloads =
-          await StorageDirectoryUtils.instance.getFilePathInAppDownloads(
-        eventId: eventId,
-        fileName: fileName,
-      );
+      String? filePathInAppDownloads;
+      if (isRoomEncrypted()) {
+        filePathInAppDownloads =
+            await StorageDirectoryUtils.instance.getDecryptedFilePath(
+          eventId: eventId,
+          fileName: fileName,
+        );
+      } else {
+        filePathInAppDownloads =
+            await StorageDirectoryUtils.instance.getFilePathInAppDownloads(
+          eventId: eventId,
+          fileName: fileName,
+        );
+      }
+
```

Comment:
what happen if this code throws an exception? only catch is enough, IMO.

Reply:
done